features_and_tools/cql.rst

```diff
@@ -0,0 +1,400 @@
+.. _cql:
+
+CQL Plugin
+============
+
+Introduction
+------------
+
+Firely Server CQL Plugin is a means to use the Firely Server to run the CQL engine. 
+
+This document demonstrates :
+    * Creation of a simple CQL file
+    * Transformation of the CQL file to a library resource using the CQL Packager Tool
+    * Setting up Firely Server CQL Plugin
+    * Getting the result of your CQL measure on a resource using Postman tool
+
+
+Create a CQL File
+-----------------
+
+To start working with Firely Server CQL Plugin, you would need a CQL File.
+An example of a CQL file ``AgeFHIR4.cql`` is displayed below: 
+
+    .. code-block:: jsonc
+
+        library AgeFHIR4 version '0.0.1'
+        using FHIR version '4.0.1'
+
+        context Patient
+
+        define "is18OrAbove":
+          AgeInYears() >= 18
+
+Generate a Library resource
+---------------------------
+
```

Comment:
Explain a bit on the library resource, why is it necessary here, and what does it do?

Reply:
Il add a link to https://www.hl7.org/fhir/library.html